app/services/editor-commands/commands/add-files.ts

```diff
@@ -61,5 +71,11 @@ export class AddFilesCommand extends Command {
 
   async rollback() {
     await this.removeNodesSubCommand.execute();
+
+    // try {
+    //   await this.removeNodesSubCommand.execute();
+    // } catch (error: unknown) {
+    //   console.error('Error rolling back add files command. ', error);
+    // }
```

Comment:
Not sure if this is intentionally being left here or just some debugging code?

Reply:
removed